cmd/ferretdb/main.go

```diff
@@ -220,7 +220,18 @@ func main() {
 	case "run":
 		run()
 	case "ping":
-		ping()
+		logger := setupLogger(cli.Log.Format, "")
+		checkFlags(logger)
+
+		ping := Ping{
+			l: logger,
+		}
+
+		ctx, _ := ctxutil.SigTerm(context.Background())
```

Comment:
The stop function should be called, not ignored. It is useful for terminating process hung during shutdown by hitting Ctrl+C / sending SIGTERM twice